plane_mcp/tools/work_item_attachments.py

```
@@ -0,0 +1,197 @@
+            project_id=project_id,
+            work_item_id=work_item_id,
+            params=params,
+        )
+
+    @mcp.tool()
+    def create_work_item_attachment(
+        project_id: str,
+        work_item_id: str,
+        name: str,
+        size: int,
+        type: str | None = None,
```

Ruff (0.15.12)

[error] Function argument `type` is shadowing a Python builtin

(A002)